README.md

```diff
@@ -29,8 +29,7 @@ subdirectory, and use functionality provided by dracut-functions to do their
 work.
 
 Documentation:
- - [Introduction](man/dracut.adoc)
- - [User Manual](man/dracut.usage.adoc)
+ - https://dracut-ng.github.io/dracut/
```

Comment:
Can this be simply `https://dracut-ng.github.io/` instead ?

Reply:
p.s. updated site is now at simply

https://people.redhat.com/~iwienand/dracut-web/